design-toolkit/components/src/components/sidenav/index.tsx

```diff
@@ -89,6 +89,26 @@ const SidenavContext = createContext<SidenavContextValue>({
   isOpen: false,
 });
 
+/**
+ * Sidenav - Collapsible side navigation panel
+ *
+ * Provides a hierarchical collapsible side navigation intended to be used
+ * inside a Drawer.Layout. Supports headers, avatars, nested menus, and items.
+ *
+ * @example
+ * <Drawer.Layout push="left">
+ *   <Sidenav>
+ *     <Sidenav.Header>
```

Comment:
I think it makes sense to flush this example out a bit more

Reply:
Updated, I pulled the code from the *.mdx file.